packages/fxa-auth-server/lib/routes/auth-schemes/hawk-fxa-token.js

```diff
@@ -89,22 +98,28 @@ function strategy(getCredentialsFunc) {
             return h.continue;
           }
 
-          const error = AppError.unauthorized('Token not found');
-          error.isMissing = true;
-          throw error;
+          throw tokenNotFoundError();
         }
 
         if (auth.toLowerCase().indexOf('hawk') > -1) {
           // If a Hawk token is found, lets parse it and get the token's id
           const parsedHeader = parseAuthorizationHeader(auth);
-          const token = await getCredentialsFunc(parsedHeader.id);
+          let token;
+
+          try {
+            token = await getCredentialsFunc(parsedHeader.id);
+          } catch (_) {
+            // we'll handle the empty token case below
+          }
```

Comment:
Previously if `getCredentialsFunc` throws an error, that's the resulting error of the auth.  This makes the error a bit more consistent, but it is a change in the API.